cmd/helm/install_test.go

```diff
@@ -78,6 +78,38 @@ func TestInstall(t *testing.T) {
 			resp:     releaseMock(&releaseOptions{name: "virgil"}),
 			expected: "virgil",
 		},
+		// Install, secrets from cli
+		{
+			name:     "install with secrets",
+			args:     []string{"testdata/testcharts/alpine"},
+			flags:    strings.Split("--set-secrets foo=bar", " "),
+			resp:     releaseMock(&releaseOptions{name: "virgil"}),
+			expected: "virgil",
+		},
+		// Install, secrets from cli via multiple --set-secrets
+		{
+			name:     "install with multiple secrets",
+			args:     []string{"testdata/testcharts/alpine"},
+			flags:    strings.Split("--set-secrets foo=bar", "--set-secrets bar=foo"),
```

Comment:
I assume you meant this to be `strings.Split("--set-secrets foo=bar --set-secrets bar=foo", " ")`?